src/viam/app/data_client.py

```diff
@@ -466,8 +469,8 @@ async def binary_data_capture_upload(
         sensor_contents = SensorData(
             metadata=(
                 SensorMetadata(
-                    time_requested=datetime_to_timestamp(data_request_times[0]) if data_request_times[0] else None,
```

Comment:
(flyby) this would have caused a runtime error in cases where the (ostensibly optional) tuple was None.